person.rb

```diff
@@ -27,4 +28,9 @@ def can_use_services?
   def correct_name
     @name
   end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.book = self
+  end
```

Comment:
- [ ] The `add_rental` method takes `book`  and `date` as  parameters, and  `person` itself is sent as a parameter to create the new rental achieving the 'has-many' association. Something like so: 

```ruby
  def add_rental(book, date)
    Rental.new(date, self, book)
  end
```

Please kindly fix that.